src/main/docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# Preliminary image that downloads and extracts Elasticsearch
+#
+FROM cafapi/opensuse-jre8:1.2 AS builder
+
+RUN zypper -n install which hostname
+RUN zypper -n install unzip
+
+# Download and Extract Elasticsearch
+ADD https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/zip/elasticsearch/2.3.3/elasticsearch-2.3.3.zip /opt/
+
+RUN unzip /opt/elasticsearch-2.3.3.zip -d /opt/
+
+# Install the analysis-icu plugin for Elasticsearch
+RUN /opt/elasticsearch-2.3.3/bin/plugin install analysis-icu
+
+#
+# The actual image definition
+#
+FROM cafapi/opensuse-jre8:1.2
+
+RUN zypper -n install which hostname
+
+# Startup Elasticsearch and expose ports
+EXPOSE 9200 9300
+CMD /opt/elasticsearch/bin/elasticsearch -Des.insecure.allow.root=true
```

Comment:
If there is a way to set this `es.insecure.allow.root` property by default in one of the config files then we should probably do that, but we can leave it for now.